test/dotnet-new3.UnitTests/DotnetNewInstantiate.cs

```diff
@@ -319,6 +320,52 @@ public void CanInstantiateTemplate_WithConditions_BasedOnFileName(string testCas
             Assert.Equal($"{string.Format(expectedCommandFormat, "bar")}{expectedEol}bar{expectedEol}", File.ReadAllText(testFile));
         }
 
+        [Fact]
+        public void CanInstantiateTemplate_ConditionalProcessing()
+        {
+            string workingDirectory = TestUtils.CreateTemporaryFolder();
+            string home = TestUtils.CreateTemporaryFolder("Home");
+            string templateName = "TemplateConditionalProcessing";
+            Helpers.InstallTestTemplate($"{templateName}/Template", _log, home, workingDirectory);
+
+            new DotnetNewCommand(_log, "TestAssets.ConditionalProcessing")
+                .WithCustomHive(home)
+                .WithWorkingDirectory(workingDirectory)
+                .Execute()
+                .Should()
+                .ExitWith(0)
+                .And.NotHaveStdErr()
+                .And.HaveStdOutContaining("The template \"ConditionalProcessing\" was created successfully.");
+
+            string expectedLocation = TestUtils.GetTestTemplateLocation($"{templateName}/Expected");
```

Comment:
this may be added to approval output instead. `Verify.NET` supports multiple verifications per test as here:
https://github.com/VerifyTests/Verify/blob/main/docs/naming.md#multiple-calls-to-verify

you may do `await Verify(actualContent).UseMethodName(fileWithExtension)` as example.
Please move the test to `.Approval.cs`.